src/iris/api.py

```diff
@@ -4253,12 +4253,12 @@ def on_get(self, req, resp):
             stats = {}
 
             for row in cursor:
-                # {statistic : {timestamp: value, timestamp: value}}
+                # format: {statistic : [{timestamp: value}, {timestamp: value}]}
```

Comment:
Why the change in format? I don't particularly care for one over the other, but curious why it changed.

Reply:
I changed it to use lists so it would preserve sorted order when rendered by the browser.